umake/frameworks/ide.py

```diff
@@ -110,24 +106,23 @@ def done(download_result):
 
             logger.debug("Preparing to download the main archive.")
             if arch == 'i686':
-                download_url = self.DOWNLOAD_URL_PAT.format(arch='', suf='')
+                download_url = self.download_page_url.format(arch='', suf='')
             elif arch == 'x86_64':
-                download_url = self.DOWNLOAD_URL_PAT.format(arch='-x86_64',
-                                                            suf='')
+                download_url = self.download_page_url.format(arch='-x86_64',
```

Comment:
Kept the naming consistent as the `download_page_url` class variable in `BaseJetBrains`